add logo macro for localised logo

Rizziepit opened the pull request.

miltontony commented on cms/templates/logo.pt:
```diff
@@ -0,0 +1,9 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal"
+     metal:define-macro="logo">
+    <div id="logo-container" tal:define="localisation view.get_localisation()" tal:condition="localisation">
+        <a href="/" tal:condition="localisation.logo_image">
```
We'll need a default logo. `/static/img/logo.png` should suffice

Rizziepit replied:
Is there a case where you'd want no logo image, only logo text?

miltontony commented on cms/templates/logo.pt:
```diff
@@ -0,0 +1,9 @@
+        <a href="/" tal:condition="localisation.logo_image">
+            <img src="${view.get_image_url(localisation.logo_image_host, localisation.logo_image)}" border="0" alt="${localisation.logo_description}"/>
+        </a>
+        <div tal:condition="localisation.logo_text">${localisation.logo_text}</div>
```
no, `logo_text` is a special case for MAMA at the moment. I'd leave it out from here and implement it on `unicore-cms-mama` override only.

Rizziepit replied:
Ok. How about `default_image_src` argument for the macro? If it's left out, nothing is rendered. Apps can then specify `default_image_src="/static/img/logo.png"` if they want.

miltontony replied:
+1 on that

Rizziepit commented:
@miltontony I updated the logo macro so that you can use it like this in a template:
```html
<div metal:use-macro="view.logo_template" tal:define="default_image_src '/default/path/to/logo.png'"></div>
```
Also added a test

Rizziepit commented:
@miltontony can I merge and make a new release?

smn commented:
Do we want a test when we fallback to the default image?

smn commented on cms/templates/logo.pt:
```diff
@@ -0,0 +1,14 @@
+            img_attrs
+        except NameError:
+            img_attrs = view.get_logo_attributes()
+    ?>
```
![bleeding](http://i.giphy.com/R7bTCHfPD9Fug.gif)

smn replied:
I know the Chameleon story here, don't worry.

smn commented on cms/templates/logo.pt:
```diff
@@ -0,0 +1,14 @@
+            <img tal:attributes="img_attrs" border="0"/>
+        </a>
+    </div>
+</div>
```
Missing new line.

smn commented:
:+1: in general and let's fast track moving away from Chameleon.
